src/book/03-guides/01-main-codebase/04-robocup-setup.mdx

```diff
@@ -0,0 +1,27 @@
+---
+section: Guides
+chapter: Main Codebase
+title: Robocup Setup
+description: Setting up the robot for robocup
+slug: /guides/main/getting-started
+authors:
+  - Ysobel Sims (@ysims)
+  - Josephus Paye II (@JosephusPaye)
+  - Mikyla Peters (@miikyla)
+  - Lachlan Court (@LachlanCourt)
+---
+
+## Networking
+
+1. `cd system/nugusX/etc/systemd/network/` to go to the config folder that system configuration will use for networkd.
+2. Rename `40-wlp58s0-robocup.network` to `30-wlp58s0-robocup.network`
+3. Rename `30-wlp58s0.network` to `40-wlp58s0.network`
+4. Double check that network, gateway and dns are correct.
+   Usually the address is `192.168.0.X` or `10.1.0.X` and both gateway and dns end with `3.1`.
+5. `cd ~` to go back to home.
+5. Add the output of `wpa_passphrase SSID PASSWORD` to `system/default/etc/wpa_supplicant/wpa_supplicant-wlp58s0.conf`
```

Comment:
This wouldn't make sense to a lot of people, particularly those running it for the first time. Might as well just say to run 
`wpa_passphrase SSID PASSWORD > system/default/etc/wpa_supplicant/wpa_supplicant-wlp58s0.conf` 
you probably should do an absolute path as well. and say what SSID and PASSWORD meanm and what the whole line does.

Reply:
If wpa_supplicant is in the docker image you could just do something with ./b shell

Reply:
I don't want people blindly appending to the `wpa_supplicant` config file, I want to make sure they look at it before they add stuff.